src/responses/stream_stats_response.ts

```diff
@@ -0,0 +1,19 @@
+import { AbstractResponse } from "./abstract_response"
+import { RawResponse } from "./raw_response"
+
+export class StreamStatsResponse extends AbstractResponse {
+  static key = 0x801c
+  readonly statistics: Record<string, bigint> = {}
```

Comment:
Is it possible to define the keys `committed_chunk_id` like for the MessageProperties (reployTo, correlationId, etc ..) ?
If it's possible, I suggest it to translate in camelcase committedChunkId

Reply:
Yes I agree. I made this change to accomodate that. https://github.com/coders51/rabbitmq-stream-js-client/pull/92/commits/ead69e5a54f30f52cbe8cadd59510ba624a75ee4